src/core_modules/capture-core/components/WidgetComment/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,164 @@
+        paddingTop: spacersNum.dp8,
+    },
+    addCommentContainer: {
+        marginRight: spacersNum.dp8,
+    },
+};
+
+const CommentSectionPlain = ({
+    notes,
+    handleAddNote,
+    classes,
+}: Props) => {
+    const [isEditing, setEditing] = useState(false);
+    const [newNoteValue, setNewNoteValue] = useState('');
+
+    const handleChange = useCallback((value) => {
+        setEditing(true);
+        setNewNoteValue(value);
+    }, []);
+
+    const onCancel = useCallback(() => {
+        setNewNoteValue('');
+        setEditing(false);
+    }, []);
+
+    const onAddNote = useCallback(() => {
+        handleAddNote(newNoteValue);
+        setNewNoteValue('');
+        setEditing(false);
+    }, [handleAddNote, newNoteValue]);
+
+    const CommentItem = ({ value, lastUpdated, lastUpdatedBy }) => (
+        <div data-test="comment-item" className={cx(classes.item)}>
+            <div className={classes.avatar} /> {/* TODO: add avatar */}
+            <div className={classes.rightColumn}>
+                <div className={classes.header}>
+                    {lastUpdatedBy && <span className={cx(classes.headerText, classes.name)}>
+                        {lastUpdatedBy.firstName} {' '} {lastUpdatedBy.surname}
+                    </span>}
+                    <span className={cx(classes.headerText, classes.lastUpdated)}>
+                        {moment(lastUpdated).fromNow()}
+                    </span>
+                </div>
+                <div className={classes.body}>
+                    <Parser>{value}</Parser>
+                </div>
+            </div>
+        </div>
+    );
+
+
+    return (
+        <div className={classes.wrapper}>
+            <div className={classes.notesWrapper}>
+                {notes
+                    .sort((a, b) => moment(a.lastUpdated).valueOf() - moment(b.lastUpdated).valueOf())
+                    .map(note => <CommentItem key={note.note} {...note} />)
+                }
+                {notes.length === 0 &&
+                    <div className={classes.emptyNotes}>
+                        {i18n.t('This event doesn\'t have any notes yet')}
+                    </div>}
+            </div>
+
+            <div className={classes.editor}>
+                <Editor>
+                    <FocusTextField
+                        placeholder={i18n.t('Write a comment about this event')}
+                        onChange={handleChange}
+                        value={newNoteValue}
+                        data-test="comment-textfield"
+                    />
+                </Editor>
+            </div>
+
+            {isEditing && <div className={classes.newCommentButtonContainer} data-test="comment-buttons-container">
+                <Button
+                    dataTest="add-note-btn"
+                    onClick={onAddNote}
+                    className={classes.addCommentContainer}
+                    primary
+                >
+                    {i18n.t('Save note')}
```

Comment:
Lets change this to 'Save comment'

Reply:
Sure will we update the design too. 
I think the `notes` still used in many place design here https://www.sketch.com/s/95c3d586-218c-4685-968b-a388270da949/a/R1ApWrl

Reply:
Yes there seems to be some inconsistencies with the wording also in the designs. Great if we can make sure the texts we use right now is "comment", then we can circle back and make sure all the docs is in line with this :)